_episodes/01-motivation.md

```diff
@@ -11,7 +11,9 @@ objectives:
   - Discuss the reasons why we advocate distributed version control.
 ---
 
-## The essence of version control
+## Motivation
```

Comment:
It seems like every heading level could be decreased by one - there's no h1 here.  Unless I'm missing something...

Reply:
For me it looks less screaming and it's easier to read but perhaps just personal preference.